tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -298,6 +298,24 @@ private void OnEndRequest(object sender, EventArgs eventArgs)
                             }
 
                             CookieAnalyzer.AnalyzeCookies(app.Context.Response.Cookies, IntegrationId);
+                            if (HttpRuntime.UsingIntegratedPipeline &&
+                                _canReadHttpResponseHeaders)
+                            {
+                                try
+                                {
+                                    HeaderAnalyzer.AnalyzeHeaders(app.Context.Response.Headers, IntegrationId.AspNetCore);
```

Comment:
This is a .net framework file. I would use the already defined IntegrationId and not IntegrationId.AspNetCore.